basics/Main.java

```diff
@@ -23,6 +24,30 @@ public static String pluralize(String word, int num){
       
   }
 
+  //Create a flipNHeads function
+  public static String flipNHeads(int num){
+
+    String counter = num > 1 ? "heads" : "head";
```

Comment:
This variable name `counter` is a little confusing since it is a String, and not actually counting anything. But I like the idea.